Core/src/org/sleuthkit/autopsy/datamodel/ContentChildren.java

```diff
@@ -22,6 +22,7 @@
 import java.util.Collections;
 import java.util.List;
 import java.util.logging.Level;
+import org.openide.util.Exceptions;
```

Comment:
Left over unused import from using Netbean's surround with try/catch option, but the PR works and seems good.